code/src/main/java/frc/robot/commands/navcommands/ResetGyro.java

```diff
@@ -20,6 +20,9 @@
     @Inject
     public ResetGyro(final ILogger logger, IDrivetrainSubsystem drivetrain) {
```

Comment:
While not typically a good idea to name classes based on a pattern. I think there are some exceptions and I think Commands and Subsystems are those. We're already suffixing Subsystems with 'Subsystem' and we have a few commands with the Command suffix. I think we should keep that consistent. So I'm asking if we can suffix commands with 'Command' :)

Reply:
@delaynieym Can provide more insight, but i was under the impression that commands and subsystems have the suffix, but nav commands didn't. 
That said, It is probably a good idea to suffix all the commands with Command.